README.md

```diff
@@ -25,23 +25,32 @@ This repository adds support for the [Windows 10 SDK](https://developer.microsof
 
 See the official [React Native website](https://facebook.github.io/react-native/) for an introduction to React Native.
 
+## System Requirements
```

Comment:
One thing that tripped me up was that the Windows Phone emulator only works on Windows 10 Pro, and not Home (because I wanted to test on both desktop and an emulated device)